kratos/python/add_utilities_to_python.cpp

```diff
@@ -284,6 +284,50 @@ void CopyModelPartNodalVarToNonHistoricalVarWithDestination(
     rVariableUtils.CopyModelPartNodalVarToNonHistoricalVar(rVariable, rDestinationVariable, rOriginModelPart, rDestinationModelPart, BuffStep);
 }
 
+template<class TDataType, class TVarType = Variable<TDataType>>
+void VariableUtilsSetVariable(
```

Comment:
what does this funciton do? a minimal comment would be needed

Reply:
It is required to export the method in Python with the same name despite the function overloading in c++.